### DISCUSSION TITLE
French government asks Waze/Google Maps to implement eco-friendly nudges

### DISCUSSION THREAD

**Discussion by breversa** [osmandapp/OsmAnd#15053] [Ideas]
https://www.ladepeche.fr/2022/08/09/voici-pourquoi-waze-plans-et-google-maps-ne-vous-proposeront-plus-le-trajet-le-plus-court-pour-arriver-a-votre-destination-10480323.php

The French government asks/demands that Waze/Google Maps implement the following nudges:
1. Display eco-friendly messages such as "For short travels, prefer walking or cycling", "Think of carpooling", "Slowing down from 130 km/h to 110 km/h on highways lowers your fuel consumption by 20%"
2. Display CO2 equivalent (https://github.com/osmandapp/OsmAnd/discussions/11943 and https://github.com/osmandapp/OsmAnd/pull/12859)
3. Display carpooling/bike renting locations from 2022-12-01 at the latest. From 2023-06-01 at the latest, also display temporary motor movement restrictions due to pollution
4. Offer 110 km/h routes over 130 km/h by default. It’s unclear if that means offering a route through 110kph roads, or through 130kph highways but with a calculated max speed of 110kph.
5. Limit alternative routes through secondary roads unless route duration is at least 10% lower

Of course, there’s no word about non-commercial/open-source software so I have no idea whether OsmAnd will have to conform to these demands (it’s unclear if these are merely suggestions or legal obligations), but I do think that some of them are good practices that should be implemented in OsmAnd as well (especially number 2 and number 4).

**Comment by breversa** [comment]
This made me think of another way to think of navigation:

Instead of asking "What’s the best way to drive/cycle/walk from point A to point B", why not ask "What’s the best way to *go* from point A to point B", and let OsmAnd define the most ecological (combination of) mean(s) of transportation.

You could of course restrict the means of transportation, as not everyone has a bike/car/motorcycle, or is even able to walk easily/for long distances.

Thoughts, anyone?

**Reply by scaidermern** [reply]
Already suggested here: #7463

The problem is, OsmAnd has not enough information to answer this question. Public transport schedules are missing, there is only limited information about rentable bicycles, scooters, cars etc.
